classification/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+# `recipe.yaml` is the main configuration file for an MLflow Recipe.
+# Required recipe parameters should be defined in this file with either concrete values or
+# variables such as {{ INGEST_DATA_LOCATION }}.
+
+# Variables must be dereferenced in a profile YAML file, located under `profiles/`.
+# See `profiles/local.yaml` for example usage. One may switch among profiles quickly by
+# providing a profile name such as `local` in the Recipe object constructor:
+# `p = Recipe(profile="local")`
+#
+# NOTE: YAML does not support tabs for indentation. Please use spaces and ensure that all YAML
+# files are properly formatted.
+
+recipe: "classification/v1"
+# Specifies the name of the column containing targets / labels for model training and evaluation
+target_col: "label"
+positive_class: 1
```

Comment:
Can we explain what positive_class means here?

Reply:
Done!